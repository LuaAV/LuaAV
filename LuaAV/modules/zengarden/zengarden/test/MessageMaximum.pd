#N canvas 802 231 450 300 10;
#X obj 46 34 loadbang;
#X obj 46 217 print;
#X obj 117 34 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 117 34 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 46 179 max;
#X msg 150 123 23;
#X msg 114 123 54;
#X msg 78 123 17;
#X obj 46 69 t b b b b b;
#X connect 0 0 8 0;
#X connect 2 0 8 0;
#X connect 4 0 1 0;
#X connect 5 0 4 0;
#X connect 6 0 4 1;
#X connect 7 0 4 0;
#X connect 8 0 4 0;
#X connect 8 1 7 0;
#X connect 8 2 6 0;
#X connect 8 3 5 0;
#X connect 8 4 4 0;
