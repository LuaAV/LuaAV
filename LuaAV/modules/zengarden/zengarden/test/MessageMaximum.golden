[@ 0.000ms] print: 0
[@ 0.000ms] print: 23
[@ 0.000ms] print: 54
[@ 0.000ms] print: 54
